testdir/REGRESS

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 uname -a
-gcc echo.c -o echo && echo echo compiled
+gcc echo.c -o echo && echo compiled echo
```

Comment:
I don't think anyone particularly cares about this output. I believe both forms are understandable. It happens that this makes my tooling happier. But I'm happy to drop it.